content/javascript/concepts/number-methods/terms/isNaN/isNaN.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.isNaN()'
+Description: 'Determine whether the passed value is NaN.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Numbers'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `Number.isNaN()` determines whether the passed value is NaN.
+
+## Syntax

```

Comment:
Could we add a little bit more here? Something like:

```markdown
The `.isNaN()` method is a part of the `Number` class in JavaScript. It accepts a single argument, `value`, and returns `true` if it is, indeed, not a number or `false`, otherwise.
```